### DISCUSSION TITLE
Feature Request: Enhance Speaker/Narrator Management for Audiobooks

### DISCUSSION THREAD

**Discussion by AndryXY** [advplyr/audiobookshelf#3516] [Ideas]
Hello everyone,

**Current Issue:**
- The term "narrator" is used to describe multiple speakers, which can be misleading.
- In German Hörspiele (audio dramas), the word "narrator" doesn't just refer to a single narrator but can also mean character speakers, similar to actors in a movie role.

**Suggested Enhancement:**
- Add a feature to support a list of characters and their respective speakers.
- Allow users to associate multiple speakers with their specific characters, similar to a movie cast list.

**Benefits:**
- Improved accuracy: Clear distinction between narrators and character speakers.
- Better discoverability: Users could look up works of a specific speaker or narrator via a "narrator page".

**Mockup:**
![narrators-001](https://github.com/user-attachments/assets/5c32fa28-aa10-4709-b413-28ac81556385)
![narrators-002](https://github.com/user-attachments/assets/49d89e9e-6b0c-4ea1-b54a-d2e0650cf669)
![narrators-003](https://github.com/user-attachments/assets/95966377-8cee-4cb1-8bc6-56c76d00e462)
![narrators-004](https://github.com/user-attachments/assets/99618ceb-652b-4477-b6ee-a67645a89a89)

Thank you for considering this improvement!

**Comment by Mad-Mart-UK** [comment]
**I think this is a great idea.** Maybe also treat the narrators like the authors and give them a profile as well (Picture etc)...
